pydatalab/src/pydatalab/mongo.py

```diff
@@ -207,4 +213,16 @@ def create_user_fts():
         db.users.drop_index(user_fts_name)
         ret += create_user_fts()
 
+    # Version control indexes
+    ret += db.item_versions.create_index("refcode", name="version refcode", background=background)
+    ret += db.item_versions.create_index("user_id", name="version user_id", background=background)
```

Comment:
See comment above about user IDs -- need to make sure we can handle multiple creators here, but I'm also not sure why we need fast querying by user (surely we always know the item ID when doing this)